### DISCUSSION TITLE
Buidl the MOVE version of Speedrun.

### DISCUSSION THREAD

**Discussion by leeduckgo** [NonceGeek/Move-Camp#314] [Move Public Goods]
Grant: `75` APT @ `NonceGeekDAO`

> https://github.com/BuidlGuidl/SpeedRunEthereum

**Comment by v1xingyue** [comment]
暂定  4.30  发布 mvp version , 包含如下内容 : 

1.  登录 使用 sui wallet 登录
2. 管理员发布内容 
3. 登录用户 submit 结果 
4. 管理员验证 submit 

完不成，就麻烦其他大佬啦。

**Reply by v1xingyue** [reply]
大家劳动节快乐。  😄  

来交作业啦。 
基本的功能已经完成: 

项目地址:  https://github.com/v1xingyue/sps-mvp 
部署地址:  https://sps-mvp.vercel.app/
